bin/changelogger/WCAdminWriteCommand.php

```diff
@@ -0,0 +1,361 @@
+<?php
+
+use Automattic\Jetpack\Changelog\ChangeEntry;
+use Automattic\Jetpack\Changelog\Changelog;
+use Automattic\Jetpack\Changelogger\Config;
+use Automattic\Jetpack\Changelogger\FormatterPlugin;
+use Automattic\Jetpack\Changelogger\Utils;
+use Automattic\Jetpack\Changelogger\VersioningPlugin;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\MissingInputException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use function Wikimedia\quietCall;
+
+/**
+ * This class was copied from Jetpack Changelogger package
+ * to override writeChangelog method() as it is not currecntly
+ * possible to extend the class.
```

Comment:
This is too bad, is this just to point at `readme.txt` instead of a `CHANGELOG` file? In either case it would be nice to extend the original class. We should create a feature request for this in the Jetpack repo, or I suppose create a PR for it ourselves.
cc: @anomiex to see if there is a preferred way?

Reply:
This is mostly for formatting the log files into our own format. If we decide to move to CHANGELOG file, we probably don't need to modify/extend the write command at all. I still need to investigate a bit.

We could also just copy this file https://github.com/Automattic/jetpack-changelogger/blob/master/bin/changelogger and load command files manually. That would allow us to extend the `write` command.

Reply:
Gotcha, in that case it's probably fine to leave it then. As long as it does get updated or extended on the changelogger side at some point.

Reply:
Sorry I didn't see the question here earlier.

You can override the filename by setting `extra.changelogger.changelog` in composer.json.

You can change the formatting to anything you want by supplying a formatter plugin, implementing the `FormatterPlugin` interface. You can even make use of a custom subclass of `Changelog` if you need to pass extra data between `parse()` and `format()`.

Is there a reason that doesn't work for you?

Reply:
@anomiex Thank you! I was not aware of `changelogger.changelog`. I will try that 👍

Reply:
@louwie17 Removed WriteCommand completely in https://github.com/woocommerce/woocommerce-admin/pull/6870/commits/3b68e723cbc032c98bb1f704a0e005cfdc4f54e4 as per @anomiex's suggestion.